operator_ui/src/pages/Notifications.test.js

```diff
@@ -2,15 +2,17 @@
 import React from 'react'
 import { Provider } from 'react-redux'
 import { MemoryRouter } from 'react-router-dom'
-import { mount } from 'enzyme'
 import configureStore from 'redux-mock-store'
 import Notifications from 'pages/Notifications'
+import { render, screen } from '@testing-library/react'
+
+const { queryByText } = screen
 
 const classes = {}
 const mockStore = configureStore()
 
 const mountNotifications = (store) =>
-  mount(
+  render(
```

Comment:
This could be trimmed if using `renderWithRouter`?

Reply:
This needs to mock a store with the notification states, which you can't do with the custom render.

I'm planning to replace these types notifications anyway, so it will be removed in the near future.